### DISCUSSION TITLE
Wired data transfer

### DISCUSSION THREAD

**Discussion by Dollar0712** [jomjol/AI-on-the-edge-device#2038] [Q&A]
### Issue

Hi is there a way to having a wired data transfer between two esp32 device?
I want to transfer the read meter data to another esp32.

**Comment by Slider0007** [comment]
Depending of what's your expextation und distance of your ESP. The UART interface would be available. Results are also published there in a limited way. You could parse the results with the other ESP.

`I (64348) POSTPROC: main: Raw: 447.1000, Value: 447.1000, Status: no error`

Why not using WiFi?

**Reply by Dollar0712** [reply]
WiFi is not available on my condition, so I am trying to use lora as my data transfer method, by using wired data transfer using pins to connect the ESP32-Cam to get the data.
I think this might be a stable way to do it.

**Reply by jonatrey** [reply]
Came here to ask the same. My gas meter and electricity meter are located outside and only get very unstable wifi connectivity. Sending values over lora or reading the values over UART would solve my problem.

**Reply by caco3** [reply]
As said, UART shows the values as well in the log. just parse it with your next device
